wordpress-mobile/WordPress-Android
# Issue/15397 comment list update

Fixes #15397

From [original](https://github.com/wordpress-mobile/WordPress-Android/pull/15715) PR by @juancoob.
Brought it up to date with trunk.

This PR fixes an issue where edits to the comment are not reflected in comment list.

To test:

- From My Site tab navigate to Comments.
- Navigate to comment details.
- From the overflow menu select Edit and edit content of the comment.
- Return to the comment list and confirm that the changes in the comment content are reflected.

## Regression Notes
1. Potential unintended areas of impact
We just call a helper method that requests all listeners to refresh content, so should be non or very little, as far as I can tell.

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing.

3. What automated tests I added (or what prevented me from doing so)
Test that the method is called at correct timing.

PR submission checklist:
